app/code/Magento/Checkout/view/frontend/web/js/action/select-shipping-address.js

```diff
@@ -12,6 +11,9 @@ define([
     'use strict';
 
     return function (shippingAddress) {
+        var address = null;
```

Comment:
Hi @Dharmeshvaja91. What the reason to set null to address variable and override it on the next line?

Reply:
okk than i direct override address variable